code/game/gamemodes/shadowling/special_shadowling_abilities.dm

```diff
@@ -155,6 +155,48 @@ var/list/possibleShadowlingNames = list("U'ruan", "Y`shej", "Nex", "Hel-uae", "N
 					M << "<span class='userdanger'>An immense pressure slams you onto the ground!</span>"
 				world << "<font size=5><span class='shadowling'><b>\"VYSHA NERADA YEKHEZET U'RUU!!\"</font></span>"
 				world << 'sound/hallucinations/veryfar_noise.ogg'
+				if(!ticker.mode.wizards)
+					message_admins("A space wizard has spawned to try and combat the Ascendant.")
+					priority_announce("ERROR \[0x0441c\]: MEMORY HIJACK EXECUTED. UNAUTHORIZED ANNOUNCEMENT DETECTED.", null, 'sound/misc/wizard_announce.ogg')
```

Comment:
Yes, complete with a new announcement sound. This gives the ascendants time for their thralls to prepare to defend it.